azure-core/azure/core/configuration.py

```diff
@@ -55,7 +57,7 @@ def __init__(self, transport=None, **kwargs):
     def get_transport(self, **kwargs):
         if self.transport:
             return self.transport(configuration=self, **kwargs)
-
+        return None
```

Comment:
Python functions will return `None` by default so this seems unnecessary. Does pylint complain here?

Reply:
yup. it does.